acceptance/custom_services_test.go

```diff
@@ -1,128 +1,110 @@
 package acceptance_test
 
 import (
-	"fmt"
-	"os"
-	"path"
-	"strconv"
-	"time"
-
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.com/suse/carrier/helpers"
 )
 
 var _ = Describe("Custom Services", func() {
 	var org = "apps-org"
 	var serviceName string
 	BeforeEach(func() {
-		serviceName = "service-" + strconv.Itoa(int(time.Now().Nanosecond()))
-
-		out, err := Carrier("create-org "+org, "")
-		Expect(err).ToNot(HaveOccurred(), out)
-		out, err = Carrier("target "+org, "")
-		Expect(err).ToNot(HaveOccurred(), out)
+		serviceName = newServiceName()
+		setupAndTargetOrg(org)
 	})
 	Describe("create-custom-service", func() {
 		It("creates a custom service", func() {
-			out, err := Carrier(fmt.Sprintf("create-custom-service %s username carrier-user", serviceName), "")
-			Expect(err).ToNot(HaveOccurred(), out)
-			out, err = Carrier("services", "")
-			Expect(err).ToNot(HaveOccurred(), out)
-			Expect(out).To(MatchRegexp(serviceName))
+			makeCustomService(serviceName)
+		})
+		AfterEach(func() {
+			cleanupService(serviceName)
```

Comment:
iiuc, `delete*` methods will raise on error while `cleanup*` versions will simply write it to stdout. Correct?

Reply:
Yes.